bearing_seal_element.py

```diff
@@ -0,0 +1,448 @@
+        ----------
+        ax : matplotlib axes, optional
+            Axes in which the plot will be drawn.
+        position : tuple
+            Position (z, y) in which the patch will be drawn.
+        Returns
+        -------
+        ax : matplotlib axes
+            Returns the axes object with the plot.
+        """
+        zpos, ypos = position
+        h = -0.75 * ypos  # height
+
+        #  node (x pos), outer diam. (y pos)
+        bearing_points = [
+            [zpos, ypos],  # upper
+            [zpos + h / 2, ypos - h],
+            [zpos - h / 2, ypos - h],
+            [zpos, ypos],
+        ]
+        ax.add_patch(mpatches.Polygon(bearing_points, color=self.color, picker=True))
+
+    @classmethod
+    def load_from_yaml(cls, n, file):
+        kwargs = load_bearing_seals_from_yaml(file)
+        return cls(n, **kwargs)
+
+    @classmethod
+    def load_from_xltrc(cls, n, file, sheet_name="XLUseKCM"):
+        kwargs = load_bearing_seals_from_xltrc(file, sheet_name)
+        return cls(n, **kwargs)
+
+
+
+class SealElement(BearingElement):
+    def __init__(
+        self,
+        n,
+        kxx,
+        cxx,
+        kyy=None,
+        kxy=0,
+        kyx=0,
+        cyy=None,
+        cxy=0,
+        cyx=0,
+        w=None,
+        seal_leakage=None,
+    ):
+        super().__init__(
+            n=n,
+            w=w,
+            kxx=kxx,
+            kxy=kxy,
+            kyx=kyx,
+            kyy=kyy,
+            cxx=cxx,
+            cxy=cxy,
+            cyx=cyx,
+            cyy=cyy,
+        )
+
+        self.seal_leakage = seal_leakage
+        self.color = "#77ACA2"
+
+    def patch(self, ax, position):
+        """Seal element patch.
+        Patch that will be used to draw the seal element.
+        Parameters
+        ----------
+        ax : matplotlib axes, optional
+            Axes in which the plot will be drawn.
+        position : tuple
+            Position in which the patch will be drawn.
+        Returns
+        -------
+        ax : matplotlib axes
+            Returns the axes object with the plot.
+        """
+        zpos, ypos = position
+        hw = 0.05
+        # TODO adapt hw according to bal drum diameter
+
+        #  node (x pos), outer diam. (y pos)
+        seal_points_u = [
+            [zpos, ypos * 1.1],  # upper
+            [zpos + hw, ypos * 1.1],
+            [zpos + hw, ypos * 1.3],
+            [zpos - hw, ypos * 1.3],
+            [zpos - hw, ypos * 1.1],
+            [zpos, ypos * 1.1],
+        ]
+        seal_points_l = [
+            [zpos, -ypos * 1.1],  # lower
+            [zpos + hw, -(ypos * 1.1)],
+            [zpos + hw, -(ypos * 1.3)],
+            [zpos - hw, -(ypos * 1.3)],
+            [zpos - hw, -(ypos * 1.1)],
+            [zpos, -ypos * 1.1],
+        ]
+        ax.add_patch(mpatches.Polygon(seal_points_u, facecolor=self.color))
+        ax.add_patch(mpatches.Polygon(seal_points_l, facecolor=self.color))
+
+
+class IsotSealElement(SealElement):
```

Comment:
Remove IsotSealElement for now. We can check later how are we going to add new bearing/seal elements.

Reply:
[Here](https://github.com/rodrigomoliveira1/ross/commit/9e5c5d12ec11d74baab8a443c61f502ee4e0369e) is the related commit for this modification.